.github/ISSUE_TEMPLATE/02-documentation.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Handsontable Documentation"
+about: Issues regarding Handsontable documentation.
+title: ""
+labels: "Type: Documentation"
```

Comment:
```suggestion
labels: "documentation"
```
If I understand correctly all concerned agreed to change the label to be compatible with wrapper labels https://github.com/handsontable/handsontable/pull/8184#discussion_r665980449.

Reply:
Done in 416f5146dfcdaff13b3291fcd5b596b9459ba68e. I would like to leave it capitalized, because that's correct.

Reply:
We need to be precise with that. As other templates use lower case in the label we need to follow that pattern.

```md
labels: "documentation"
```

Reply:
Yes, but most of the labels are capitalized. Maybe a better solution would be to correct these template labels in a way that all of them were capitalized and linguistically correct?

<img width="296" alt="Screenshot 2021-07-08 at 12 20 59" src="https://user-images.githubusercontent.com/30186818/124906283-3126d500-dfe7-11eb-8be2-7935872c5805.png">

Reply:
Does it matter? So if we decide to capitalize labels we will have to update templates?

I'm asking because I don't know how strict is GitHub when searching for matching labels

Reply:
For GH is doesn't, but it looks neater on the issue board as all labels are in the 1st convention and linguistically correct.

Reply:
The letter case matter. See the link that creates an issue https://github.com/handsontable/handsontable/issues/new?assignees=&labels=Angular&template=02-handsontable-angular.md&title=. The `labels=` param contains `Angular` label that is not automatically added to the form.

Reply:
Not all labels are capitalized. It's not for us to decide. I consulted CS and all projects are lowercase

Reply:
Lower case in d48bf66d4989b7b3b6133ecea2627e94e2cd2b6a.